src/libstd/os/linux/raw.rs

```diff
@@ -155,6 +155,11 @@ mod arch {
     }
 }
 
+#[cfg(target_arch = "mips64")]
+mod arch {
+    pub use libc::{off_t, ino_t, nlink_t, blksize_t, blkcnt_t, stat, time_t};
```

Comment:
Not sure if I need to add stability attributes. This whole module is deprecated anyway.